docs/faqs/NuGet-FAQ.md

```diff
@@ -232,16 +232,17 @@ If that fails, try the [tips mentioned in this StackOverflow post](http://stacko
 
 ## What is the API endpoint for nuget.org?
 
-### Strongly Recommended
-- For the vast majority of uses, we strongly recommend you access nuget.org via the v3 protocol uri.
-- V3: `https://api.nuget.org/v3/index.json`
+To use  nuget.org as a package repository with NuGet clients, you would need to use the following V3 API endpoint: 
 
-### Useful for NuGet Clients before 3.0 or *NuGet.exe List* Users
+**`https://api.nuget.org/v3/index.json`**
+
+### V2 API endpoint for older 3.0 NuGet clients
 - Older clients can still use the v2 protocol to reach nuget.org. However, please note, NuGet clients 3.0 or later will have slower and less-reliable service using the v2 protocol.
-- *NuGet.exe List* only works with the v2 protocol.
 - V2: `https://www.nuget.org/api/v2/` (DEPRECATED!!!)
 - Note: the "www." is important.
 
+- *NuGet.exe List* only works with the v2 protocol.
```

Comment:
> Additionally, NuGet.exe List only works with the v2 protocol.

This is misleading. NuGet.exe list works when you point NuGet.exe to the V3 source. Under the covers, it happens to use the V2 protocol similar to push/delete... but this is an implementation detail.

What is trying to be communicated here?